src/conditions/requiredPath.ts

```diff
@@ -0,0 +1,35 @@
+import { ConditionConfig } from '../config'
+import { PullRequestInfo } from '../models'
+import { ConditionResult } from '../condition'
+
+export default function hasRequiredPaths (
+  config: ConditionConfig,
+  pullRequestInfo: PullRequestInfo
+): ConditionResult {
+  if (!config.requiredPathRegex) {
+    return {
+      status: 'success'
+    }
+  }
+  if (pullRequestInfo.changedFiles > 100) {
+    return {
+      status: 'fail',
+      message: 'requiredPath condition supports maximum 100 files per one PR at this moment. Sorry.'
+    }
+  }
```

Comment:
Unfortunately, I can't get this _pagination_ thing, so I had to add this condition.
If you could get a rid of it, it would be awesome!